lib/tier-node.js

```diff
@@ -73,6 +73,7 @@ function handler (handle, params, req, res, body, query) {
   if (!handle) {
     res.statusCode = 404
     res.end()
+    return
```

Comment:
this sounds like a bug. Would you mind adding a test for this? Or was it covered by ones added in this PR?